wagtail/contrib/frontend_cache/utils.py

```diff
@@ -146,6 +146,112 @@ def purge_pages_from_cache(pages, backend_settings=None, backends=None):
         purge_urls_from_cache(urls, backend_settings, backends)
 
 
+def purge_site(
```

Comment:
Suggestion: A `purge_all` util would be quite helpful here too. 

What happens if the site is the default site? In those cases, the `hostname` isn't necessarily representative, so purging everything may be the only way forward.

Reply:
@RealOrangeOne Where a project did need to purge 'everything' - The expectation would be that the implementor do:

```
for site in Site.objects.all():
    purge_site(site)
```

This would mean that `purge_hostname` would be used where available, and `purge_all()` would only be used in cases where `purge_hostname` isn't a possibility